Cargo.toml

```diff
@@ -2,14 +2,14 @@
 name = "xmas-elf"
 version = "0.2.0"
 authors = ["Nick Cameron <ncameron@mozilla.com>"]
-description = "Library for parsing and navigatin ELF data; zero-allocation, type-safe."
-license = "Apache-2.0/MIT"
+description = "Library for parsing and navigating ELF data; zero-allocation, type-safe."
+license = "Apache-2.0 OR MIT"
 repository = "https://github.com/nrc/xmas-elf"
+keywords = ["elf", "parser", "binary"]
 
 [dependencies]
 zero = "0.1"
 
 [[bin]]
 name = "xmas_elf"
-publish = "false"
```

Comment:
So, the first time I published this to crates.io, it used src/bin/main.rs as the crate (iirc) and it was useless. So I added this line and then publish published the lib (see this commit - https://github.com/nrc/xmas-elf/commit/5e2cc78d33ecca9de7c2edb16af7d384a6e0df82). This may well be horrible cargo culting (no pun intended), I just did what someone on irc told me rather than look into it properly.